crosstool/setup.bzl

```diff
@@ -1,5 +1,6 @@
 """Configure the Apple CC toolchain"""
 
+load("@bazel_features//:features.bzl", "bazel_features")
```

Comment:
We'll need to add this to `MODULE.bazel`

Reply:
Or i guess its complaining in `WORKSPACE.bazel` when not using bzlmod, since its already in the bzlmod file.